components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -30,23 +30,28 @@ typedef NS_ENUM(NSUInteger, MDCTabBarViewLayoutStyle) {
    the leading edge of the bar. */
   MDCTabBarViewLayoutStyleScrollable = 1,
 
+  /**
+   The same as MDCTabBarViewLayoutStyleScrollable, but the selected tab is centered within the bar
+   if its position in the scrollview's content area permits it.*/
```

Comment:
Nit: New line

Reply:
This is just being consistent with all the other cases. Clang format didn't have an issue.